pdr_backend/pdr_dashboard/callbacks/callbacks_home.py

```diff
@@ -187,6 +192,13 @@ def update_feeds_table_on_search(
             predictoor_feeds_only, predictoors_addrs, search_value, selected_feeds
         )
 
+        if (
+            stored_predictoors_addrs
```

Comment:
what is the logic of this? Can you explain it here or in a comment? Please also note that selected_feeds is a list of dictionaries (table data), while your assignation is a list of ints, so the change is improper to the nature of the variable. Are you instead looking to change selected_feed_indices instead?

Reply:
Basically what that block does is to select all feeds of the stored predictoors. We want to do this only on the initial load that's why the `sorted(predictoors_addrs) == sorted(stored_predictoors_addrs)` check, when user selects other predictoors we don't want to automatically select the feeds